src/utils/auth.ts

```diff
@@ -0,0 +1,26 @@
+import axios, { AxiosError } from 'axios';
+import Environment from '../Environment';
+
+export function er401Feil(error: any) {
+  return error && error.response && error.response.status === 401;
+}
+
```

Comment:
Det er her det skjer, og det jeg er mest usikker på. Vi følger med på alle responser som mottas fra axios, dersom det er en 401 redirecter vi til loginservice. Blir dette for enkelt? Blir det en flaskehals? Klarer vi fange opp alle feil?
Burde vi gå for passportløsing, eller liknende (som ks-sak frontend)? 

Klarer vi mellomlagring - gjenoppgygging av søknad?

Reply:
brukAutentisering: any,
  settAutentisering: any
:any? Noen bedre forslag?